package.json

```diff
@@ -1,137 +1,40 @@
 {
-  "name": "spectacle",
```

Comment:
New root `package.json` file – mostly just here for some repo-wide dev deps (like ESLint, prettier, etc.) and to run some root scripts. Do not need any npm-specific package details, since we don't release from this `package.json` – the file details will like in `packages/**/package.json` files.